### ISSUE TITLE
Layout break - content overflows container in Yahoo iOS (& Android) app

### ISSUE THREAD

**Issue by zebulonedwardbubbel** [foundation/foundation-emails#681]
I've tested builds of the _marketing_ and _sidebar-hero_  f4e responsive email templates on Yahoo's iOS and Android apps, but the anomaly affects other templates too.

The flow content on the Yahoo iOS app spreads beyond the limits of .container, .wrapper-inner & wrapper elements in mobile view, as you can see below:
 
![img_0046](https://cloud.githubusercontent.com/assets/6442920/22105418/2acef894-de44-11e6-8f40-002e28c3c93a.PNG)
![img_0047](https://cloud.githubusercontent.com/assets/6442920/22105419/2acfbd4c-de44-11e6-9f3d-303f3be3493a.PNG)
![img_0048](https://cloud.githubusercontent.com/assets/6442920/22105421/2ad28496-de44-11e6-986f-95003f9377ea.PNG)
![img_0049](https://cloud.githubusercontent.com/assets/6442920/22105420/2ad0a98c-de44-11e6-8ab7-ae96cdbaf2e4.PNG)

If the media query rules are embedded in the \<body\> instead of \<head\> (according to this Litmus community discussion: https://litmus.com/community/discussions/5836-yahoo-app-no-longer-responsive#comment-9435), the same behaviour is observable on Yahoo's Android app too.

**Comment by markadrake** [comment]
You are suggesting that the media query rules be injected in the `body` of the HTML instead of `head` based on the discussion in the Litmus community?

Is there any official word from Yahoo about this regression and if it was on purpose? I'm not aware of an official communication channel for the dev team there.

If it's purposeful, we should try updating it to put the styles into the body of the content and see if there are any issues in this change with other email web and native apps.

> Interesting tidbit I learned today - parts of gmail now use embedded css. [From Litmus](https://litmus.com/community/discussions/6116-here-s-why-litmus-didn-t-inline-css-for-its-first-newsletter-of-2017).

**Comment by zebulonedwardbubbel** [comment]
Correct, I simply found that the style rules embedded in the `head` are totally ignored by the Yahoo mobile apps, but when I put the styles in the `body`, they are not neglected. Then I read the discussion cited above which confirmed my test results. I haven't any official information from Yahoo about this issue either.
Furthermore, I think that to put the `<style>` in the `body` would be counter-intuitive and unsuitable as the right direction should be to embed styles the standard way (in the `head`).
Anyway, I tried to point out that the email templates built with the Foundation for Emails 2 framework break on these clients, but I couldn't find the exact reasons and solve this problem.

**Comment by markadrake** [comment]
I've [submitted a suggestion at UserVoice for Yahoo](https://yahoo.uservoice.com/forums/210695-yahoo-mail/suggestions/17839918-allow-styles-and-stylesheets-in-the-head-of-an-htm). This is the only way I know to even send a question to them and get any response. Feel free to help vote it up, or add to the discussion there. Maybe Yahoo will see it and respond.

I agree it's not the ideal situation - but what about writing HTML emails has been ideal? If a good amount of your subscribers / users / mailing list use Yahoo - and Yahoo strips the styling information from the `<head>` element, that's a good case for making the switch. Ultimately - _I hope that Yahoo responds and reverts this change._

**Update:**

Yahoo has recognized the issue and it's been updated to "gathering feedback". [I think it would be helpful for people to jump into the conversation](https://yahoo.uservoice.com/forums/210695-yahoo-mail/suggestions/17839918-allow-styles-and-stylesheets-in-the-head-of-an-htm?tracking_code=fe3aebaa1779c9d14d1a0bd9cee9db37). Thanks!

**Comment by zebulonedwardbubbel** [comment]
I think it worth to mention this workaround presented by Ben Wakeford in the Foundation Forum: http://foundation.zurb.com/forum/posts/50821#comment_37118

**Comment by davehoran** [comment]
@markadrake "but what about writing HTML emails has been ideal? " -- agreed. lol

**Comment by joeworkman** [comment]
Foundation for Emails has been superseded by [Inky v2](https://github.com/foundation/inky), a complete rewrite in Rust with improved Outlook support, hybrid output mode, bulletproof buttons, validation, and more. This issue will not be addressed in the legacy codebase.

If this issue persists in v2, please open a new issue at https://github.com/foundation/inky/issues
